data/data.go

```diff
@@ -74,15 +159,21 @@ func (y EffxYaml) isEffxYaml() bool {
 
 func (y EffxYaml) newConfig() (*effx_api.ConfigurationFile, error) {
 	config := &effx_api.ConfigurationFile{}
+
 	yamlFile, err := ioutil.ReadFile(y.FilePath)
 	if err != nil {
 		return nil, err
 	}
 
+	relativePath, err := convertToRelativePath(y.FilePath)
+	if err != nil {
+		relativePath = y.FilePath
+	}
+
 	config.FileContents = string(yamlFile)
 	config.SetAnnotations(map[string]string{
 		"effx.io/source":    "effx-cli",
-		"effx.io/file-path": y.FilePath,
+		"effx.io/file-path": relativePath,
```

Comment:
It would be cool if we had another annotation to store the repo name or git URL. The path in combination with a URL will help us link to the relevant file in source control.

Reply:
i'll add a repo name, but can I add the git url in the next pr? I mean if you want I can do it here it's just that I want to make incremental progress

Reply:
Yeah a follow-up is fine!